### PR TITLE
[LOADTESTSERVICE] Adding Changes as per APIView recommendations

### PR DESCRIPTION
**Pull request by NiveditJain**

### CODE REVIEW

File: sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py [hunk 1]
```diff
@@ -45,12 +45,12 @@
     build_test_run_create_or_update_server_metrics_config_request,
     build_test_run_delete_test_run_request,
     build_test_run_get_app_components_request,
+    build_test_run_get_metric_definitions_request,
```
Comment (iscai-msft) [inline sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py:48]: can you add a changelog for these changes?

File: sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_patch.py [hunk 2]
```diff
@@ -182,6 +181,76 @@ class to perform operations on TestRun
...
+    async def begin_test_run(
+        self,
+        test_run_id: str,
+        body: JSON,
+        *,
+        poll_for_test_run_status=True,
+        old_test_run_id: Optional[str] = None,
+        content_type: str = "application/merge-patch+json",
+        **kwargs: Any
+    ) -> AsyncLoadTestingLROPoller:
```
Comment (iscai-msft) [inline sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_patch.py:194]: you should include in brackets what the poller will return, i.e. `AsyncLoadTestingLROPoller[JSON]` (not sure that's what this fn actually returns, this is more of an example

File: sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py (hunk 1 above)
Comment (NiveditJain) [reply sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py:48]: added with the latest commit

**Review by iscai-msft — APPROVED** [review]
approving bc my comments are all doc related

File: sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/_generated/aio/operations/_patch.py [hunk 3]
```diff
@@ -205,8 +261,6 @@ async def begin_test_run(
          provided, the test will run with the JMX file, configuration and app components from the
          existing test run. You can override the configuration values for new test run in the request
          body. Default value is None.
-        :param poll_for_test_run_status: If true, polls for test run status, else does not
```
[the remark this answers is not in the document]
Comment (NiveditJain) [reply sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/_generated/aio/operations/_patch.py:208]: yes, this has been removed as per the APIView comments

**Pull request merged**